examples/nix/default.nix

```
@@ -0,0 +1,98 @@
+{ inputs
```

Maybe use `nix-src` rather than the whole `inputs` attrset (the same for lix)?
```suggestion
{ nix-src
```